trunner/harness/plo.py

```diff
@@ -112,7 +118,9 @@ def send_cmd(self, cmd: str):
             self.dut.send(cmd + "\r")
             self.dut.expect_exact(cmd)
         except pexpect.TIMEOUT as e:
-            raise PloError("failed to read echoed command", cmd=cmd, output=self.dut.before) from e
+            raise PloError(
```

Comment:
Please place black corrections in the separate commit - it will be misleading when you click on those lines and see `add sleep` commit.

Reply:
Autoformat collected this, sadly (Not done on purpose)